gomint-server/src/main/java/io/gomint/server/network/PlayerConnection.java

```diff
@@ -910,7 +911,13 @@ void close() {
         LOGGER.info( "Player {} disconnected", this.entity );
 
         if ( this.entity != null && this.entity.getWorld() != null ) {
-            this.networkManager.getServer().getPluginManager().callEvent( new PlayerQuitEvent( this.entity ) );
+            PlayerQuitEvent event = this.networkManager.getServer().getPluginManager().callEvent( new PlayerQuitEvent( this.entity ) );
+            event.setQuitMessage( ChatColor.YELLOW + this.entity.getDisplayName() + ChatColor.YELLOW + " left the game." );
+            if( event.getQuitMessage() != null && event.getQuitMessage() != "" ) {
```

Comment:
In java, never ever compare string objects by identity. Use `!event.getQuitMessage().isEmpty()` instead here in the second part of the if.

Reply:
Since when?